src/main/java/com/puppycrawl/tools/checkstyle/JavadocPropertiesGenerator.java

```diff
@@ -126,11 +126,12 @@ private static void iteratePublicStaticIntFields(DetailAST objBlock, Consumer<St
             throws CheckstyleException {
         for (DetailAST member = objBlock.getFirstChild(); member != null;
                 member = member.getNextSibling()) {
-            if (isPublicStaticFinalIntField(member)) {
-                final DetailAST modifiers = member.findFirstToken(TokenTypes.MODIFIERS);
+            boolean isVariables = member.getType() == TokenTypes.VARIABLES;
```

Comment:
Some change is needed here, since `member` is now a `VARIABLES` and not a `VARIABLE_DEF`; `isPublicStaticFinalIntField` will always return false.  This is one of the reasons that `tokentypes.properties` was empty.

Reply:
ok, thanks. Now it is clear.